src/Backend/Core/Engine/DataGridFunctions.php

```diff
@@ -50,12 +50,12 @@ public static function cleanupPlainText($var)
     /**
      * Format a number as a float
      *
-     * @param float $number   The number to format.
-     * @param int   $decimals The number of decimals.
+     * @param float $number The number to format.
+     * @param int $decimals The number of decimals.
      *
      * @return string
      */
-    public static function formatFloat($number, $decimals = 2)
+    public static function formatFloat(float $number, int $decimals = 2): string
```

Comment:
Only if you use the strict option, and we don't use that, this way php will first try to convert it to the correct type
```
php > $bob = function(float $myNumber) {
php { var_dump($myNumber);
php { };
php > $bob(1);
php shell code:2:
double(1)
```